autoload/db_ui/schemas.vim

```diff
@@ -145,14 +148,19 @@ function! db_ui#schemas#get(scheme) abort
   return get(s:schemas, a:scheme, {})
 endfunction
 
-function! db_ui#schemas#query(db, query) abort
-  let base_query = db#adapter#dispatch(a:db.conn, 'interactive')
-  return map(
-  \   systemlist(
-  \     a:db.scheme ==# 'oracle' ?
-  \       printf('%s %s', a:query, base_query) :
-  \       printf('%s %s', base_query, a:query)
-  \   ),
-  \   'substitute(v:val, "\r$", "", "")'
+function! db_ui#schemas#format_query(db, scheme, query) abort
```

Comment:
There's no need for this function to be public. Make it a script level function since it's used only in this file.